### PR TITLE
Stacks

### PR DESCRIPTION
**Pull request by atriadhiakri2000**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [ Y ] I've read the contribution guidelines.
- [ Y ] I've checked the issue list before deciding what to submit.
- [ Y ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by atriadhiakri2000** [comment]
@atarax665 @plazzy99 @iamrajiv  Please check

**Review by vatsalkesarwani12 — APPROVED** [review]
👍🏻 
Always, Try adding 1 pr for 1 issue

**Comment by atriadhiakri2000** [comment]
> 👍🏻
> Always, Try adding 1 pr for 1 issue

All right Bro  👍🏻

File: Python/other/Check_for_balanced_parentheses.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+''' 
+Check-for-balanced-parentheses
+Problem Statement : Given an expression string x. Examine whether
+                    the pairs and the orders of
+                    “{“,”}”,”(“,”)”,”[“,”]” are correct in exp.
+                    For example, the function should return
+                    'true' for exp = “[()]{}{[()()]()}” and 'false' for exp = “[(])”.
+                    
+Input : Enter list of parentheses.
+
```
Comment (atarax665) [inline Python/other/Check_for_balanced_parentheses.py]: One blank line is enough, add the problem link from any coding website too.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@atriadhiakri2000 Your codes are not following PEP 8 standard. Format all your code according to PEP 8 standard.

File: Python/other/Check_for_balanced_parentheses.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+#INPUT
+
+# Enter the list of the brackets
+brackets = input("Enter the brackets: ")
+
+#OUTPUT
+
+if(Check(brackets) == True):
+    print("The parentheses are balnced")
```
Comment (atarax665) [inline Python/other/Check_for_balanced_parentheses.py:53]:
```suggestion
    print("The parentheses are balanced")
```

File: Python/other/Check_for_balanced_parentheses.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+else:
+    print("The parentheses are not balnced")
```
Comment (atarax665) [inline Python/other/Check_for_balanced_parentheses.py:55]:
```suggestion
    print("The parentheses are not balanced")
```

File: Python/other/Check_for_balanced_parentheses.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+'''
+SAMPLE INPUT/OUTPUT:
+
+INPUT
+
+    Enter the brackets: {{[[[()]]]}}{(())}
+
+OUTPUT
+
+    The parentheses are balnced
```
Comment (atarax665) [inline Python/other/Check_for_balanced_parentheses.py]:
```suggestion
    The parentheses are balanced
```

File: Python/other/Check_for_balanced_parentheses.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    
+'''
+
```
Comment (atarax665) [inline Python/other/Check_for_balanced_parentheses.py]: Remove this extra line break.

File: Python/other/Next_Greater_Element.py [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+#OUTPUT
+
+NGE(numbers,inputs)
+
+'''
+SAMPLE INPUT/OUTPUT:
+
+INPUT
+
+    Enter the brackets: {{[[[()]]]}}{(())}
```
Comment (atarax665) [inline Python/other/Next_Greater_Element.py:50]: Update the Sample I/O.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by atriadhiakri2000** [comment]
@iamrajiv Please check

**Review by atarax665 — CHANGES_REQUESTED** [review]
Move ```Implement_Queue_using_Stack.py``` and ```Implement_Stack_using_Queue.py``` to the ```ds folder```. Do the suggested changes ASAP so that I can merge  this PR.

**Comment by HarshCasper** [comment]
Closing the PR because of no updates.

**Pull request closed without merging**